sql/src/main/java/io/crate/executor/transport/task/elasticsearch/ESGetTask.java

```diff
@@ -96,8 +97,29 @@ public ESGetTask(UUID jobId,
             extractors.add(SYMBOL_TO_FIELD_EXTRACTOR.convert(symbol, ctx));
         }
 
-        final FetchSourceContext fsc = new FetchSourceContext(ctx.referenceNames());
+        boolean fetchSource = false;
+        List<String> includes = new ArrayList<String>();
 
+        for (Reference ref : ctx.references()) {
+            if (ref.ident().columnIdent().isSystemColumn()) {
+                if (ref.ident().columnIdent().name().equals("_raw")
+                        || ref.ident().columnIdent().name().equals("_doc")) {
+                    fetchSource = true;
+                    break;
+                }
+            } else {
+                includes.add(ref.ident().columnIdent().name());
+            }
+        }
+
+        String[] includeRefNames = new String[includes.size()];
```

Comment:
this is only needed if includes.size()>0 and fetchSource = false
add a conditional branch below